docs/reference/networking.md

```diff
@@ -0,0 +1,51 @@
+# 5G Network Architecture
+
+5G was built with network functions divided by services which is known as the 5G core Service-Based Architecture (SBA). The 5G core is a network of interconnected services, as illustrated in the figure below.
+
+```{image} ../images/5g_network_architecture.png
+:alt: 5G Network Architecture
+:height: 600
+:align: center
+```
+
+The 5G Core Network is composed of various network functions, each serving a unique purpose. These functions communicate internally and externally over well-defined standard interfaces.
+The User Plane Function is a critical component of the 5G core network architecture. It oversees the management of user data during the data transmission process. The UPF serves as a connection point between the RAN and the data network.
+UPF Interfaces/reference points with employed protocols:
+- Access (N3): Interface between the RAN (gNB) and the UPF
+- Core (N6): Interface between the Data Network (DN) and the UPF
+- K8s LoadBalancer (N4): Interface between the Session Management Function (SMF) and the UPF  
+
+
+A minimal 5G network setup is given in the figure below.
+
+```{image} ../images/5g_network_diagram.png
+:alt: 5G Network Diagram
+:height: 600
+:align: center
+```
```

Comment:
Do we really need the two images? It feels like information could be combined in only 1

Reply:
This diagram is added just to makes the IP addresses meaningful. This diagram needs to be removed together with the ip addresses. First image includes the logic that we are trying to give.

Reply:
Done